backend/scripts/add_admin_user.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ADMIN_USERNAME=${1}
+ADMIN_EMAIL=${2}
+ADMIN_PASSWORD=${3}
+
+docker exec -it amandus_backend_1 yarn add-admin $ADMIN_USERNAME $ADMIN_EMAIL $ADMIN_PASSWORD
```

Comment:
```suggestion
sudo docker exec -it amandus_backend_1 yarn add-admin $ADMIN_USERNAME $ADMIN_EMAIL $ADMIN_PASSWORD
```
Would add sudo here since many users (like me) cannot run docker without sudo

Reply:
Good idea, now running scripts prompts root authentication